docs/releases/v1.6.0.md

```diff
@@ -106,18 +110,19 @@ $ kubectl get --export -o=json clusterissuer | \
             .metadata.creationTimestamp,
             .metadata.annotations["kubectl.kubernetes.io/last-applied-configuration"],
             .metadata.generation)' > cert-manager-backup-clusterissuer.json
-
+# omitted output
 $ kubectl get -o json --export=true \
     --all-namespaces \
     issuer,certificates,certificaterequests  | \
     jq '.items[] |
-	del(.status,
+  del(.status,
```

Comment:
It's not missinng the `\` in line :117 ? 

I did like the previous indentation

Reply:
the ' makes it work without \